src/CONST.ts

```diff
@@ -1283,6 +1283,17 @@ const CONST = {
         SYNC: 'sync',
         IMPORT_CUSTOMERS: 'importCustomers',
         IMPORT_TAX_RATES: 'importTaxRates',
+        IMPORT_TRACKING_CATEGORIES: 'importTrackingCategories',
+        MAPPINGS: 'mappings',
+        TRACKING_CATEGORY_PREFIX: 'trackingCategory_',
+        TRACKING_CATEGORY_FIELDS: {
+            COST_CENTERS: 'cost centers',
+            REGION: 'region',
+        },
+        TRACKING_CATEGORY_OPTIONS: {
```

Comment:
These shouldn't be hardcoded, right?

Reply:
@lakchote I had asked if these will be hardcoded. Thread [here](https://expensify.slack.com/archives/C036QM0SLJK/p1714638671089049?thread_ts=1713799302.291019&cid=C036QM0SLJK).

Reply:
These will be hardcoded yes.